TundraTD/Assets/Scripts/Mobs/MobPortal.cs

```diff
@@ -9,26 +10,48 @@ public class MobPortal : MonoBehaviour
         [SerializeField] private Transform gates;
         [SerializeField] private MobWave[] mobWaves;
         [SerializeField] private Transform mobSpawner;
+        public int _allMobs;
         
         private void Update()
         {
             if (Input.GetKeyDown(KeyCode.B))
                 SpawnNextMob();
         }
-
+        public void DecreaseMobsCountByOne()
+        {
+            _allMobs -= 1;
+        }
         public void SpawnNextMob()
         {
+            StartCoroutine(IntervalsBetweenWaves());
+        }
+
+
+        IEnumerator IntervalsBetweenWaves()
+        {
+            bool _firstWave = true;
             foreach (var mobWave in mobWaves)
             {
+                while (_allMobs != 0)
+                {
+                    yield return new WaitForSeconds(1);
+                }
+                if (!_firstWave)
+                {
+                    yield return new WaitForSeconds(5);
+                }
+                _firstWave = false;
                 foreach (var mobProperty in mobWave.MobProperties)
                 {
                     for (int i = 0; i < mobProperty.MobQuantity; i++)
                     {
+                        yield return new WaitForSeconds(1.5f);
```

Comment:
Extract the magical numbers `1.5` from the function in [SerializeField] variable